crypto/x509/v3_purp.c

```diff
@@ -436,17 +459,17 @@ int x509v3_cache_extensions(X509 *x)
             || X509_get_ext_by_NID(x, NID_issuer_alt_name, -1) >= 0) {
             x->ex_flags |= EXFLAG_INVALID;
         }
-        if (pci->pcPathLengthConstraint) {
+        if (pci->pcPathLengthConstraint != NULL)
             x->ex_pcpathlen = ASN1_INTEGER_get(pci->pcPathLengthConstraint);
-        } else
```

Comment:
The `{` doesn't seem needed here.

Reply:
Yes, and that's why I removed it, also for adherence to the coding guidelines.

Reply:
Clearly I was confused.